docs/tutorials/learn.md

```diff
@@ -725,7 +725,7 @@ type Maybe (A : Type) :=
 For example, one could define the tail function as:
 
 ```juvix
-tail' : {A : Type} -> List A -> Maybe A
+tail' : {A : Type} -> List A -> Maybe (List A)
```

Comment:
Oh, good catch! in the future, we'll type-check the examples to avoid this kind of mistakes.